troubleshoot/deployments/cloud-enterprise/rebuilding-broken-zookeeper-quorum.md

```diff
@@ -67,29 +67,46 @@ Perform the following steps on each host to back up the Zookeeper data directory
 
 ## Determine the Zookeeper leader [ece_determine_the_zookeeper_leader] 
 
-If a Zookeeper quorum is broken, you must establish the best Zookeeper leader to use for recovery before you start the recovery proces.
+If a Zookeeper quorum is broken, it's crucial to identify the best Zookeeper leader candidate to use for recovery before you start the recovery proces.
 
-The simplest way to check is using the [Zookeeper sync status](verify-zookeeper-sync-status.md) command.
+Collect the following Essential Information (A) and (B) from **all ECE director hosts that have ZK containers running**. This includes any recently created or decommissioned hosts.  Then reach out to [Elastic Support](/troubleshoot/index.md#contact-us).
```

Comment:
suggest making this a little more structured. "Essential Information (A) and (B)" doesn't really fit our tone - to ensure that EVERYTHING you need is collected (which I assume is your goal), I suggest approaching this as a list of things to gather before contacting support.

```suggestion
Collect the following information from all ECE director hosts that have ZK containers running, including any recently created or decommissioned hosts. After you have gathered the information, reach out to [Elastic Support](/troubleshoot/index.md#contact-us) to identify the best ZK leader candidate.

* [Output of file list and sizes of Zookeeper directories](#zk-file-list-sizes)
* [ECE diagnostics](#ece-diagnostics)
```